src/main/groovy/com/google/protobuf/gradle/ProtobufExtract.groovy

```diff
@@ -109,11 +110,11 @@ class ProtobufExtract extends DefaultTask {
 
   protected void setDestDir(File destDir) {
     Preconditions.checkState(this.destDir == null, 'destDir already set')
-    this.destDir = destDir
-    outputs.dir destDir
+    this.destDir = project.files(destDir)
+    outputs.dirs(this.destDir)
   }
 
   protected File getDestDir() {
-    return destDir
+    return destDir.getSingleFile()
```

Comment:
The changes in this file doesn't seem necessary.

Reply:
Yes, you are right.